optuna/serialization.py

```diff
@@ -0,0 +1,20 @@
+import pickle
+
+import optuna
+
+
+def save_study(study: optuna.study.Study, path: str) -> None:
+    study_in_memory = optuna.create_study(directions=study.directions)
+    for trial in study.trials:
```

Comment:
How about copying `study.user_attrs` and `study.system_attrs` too?

Reply:
Makes sense to include in the serialization. Thanks, added!

Reply:
Thanks!